crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net.rs

```diff
@@ -897,23 +895,18 @@ impl MetaNet {
         let spawn_host = host.clone();
         tokio::task::spawn(async move {
             while let Some(evt) = ep_evt.recv().await {
-                let evt = match evt {
-                    Ok(evt) => evt,
-                    Err(err) => {
-                        tracing::error!(?err, "tx5 err event");
-                        continue;
-                    }
-                };
-
                 match evt {
-                    tx5::EpEvt::Connected { rem_cli_url } => {
+                    tx5::Ep3Event::Error(err) => {
+                        panic!("Fatal tx5 error: {err:?}");
```

Comment:
While Holochain is becoming less network dependent, I'm asking can Holochain continue functioning with local only data despite this failure?

Reply:
not while it's a panic : )

Reply:
I just double checked the code, this can only happen in the case of a code mistake, so it is correct to fail fast and let us know something was written incorrectly.

Reply:
I changed my mind. I switched it to a tracing error after all: https://github.com/holochain/holochain/pull/3287/commits/63250c930e1b75ccb4b7dc00100e8dccd3a2ade3

Reply:
Huh, how come?

Reply:
I don't think it's possible to say a transient error will never in the future be introduced into this path... It would be a problem if some naive change in the future makes it so conductors crash every time a connection drops or something.